### ISSUE TITLE
Use of pagination and logic to fill homepage sections

### ISSUE THREAD

**Issue by chrismplus** [mplusmuseum/mplusmuseum-stories#131]
Hi @rdunk -- watching people browse around the site today in different viewports. I'm seeing instances like this popping up: 
![homepage-logic](https://user-images.githubusercontent.com/26731347/33162685-cff3fb4e-d065-11e7-95b2-f475d7384cd7.jpg)

Can we introduce some logic to solve this odd layout issue? Like, if there's content available (tiles), fill to the pagination button? Let me know whether you have any ideas on this???

**Comment by rdunk** [comment]
Could be a tricky one but will certainly look into this.

**Comment by chrismplus** [comment]
Updated title on this one for clarity.

**Comment by chrismplus** [comment]
Moving this one to the front of the queue.

**Comment by rdunk** [comment]
Implemented and live. Works like this:

The max number of items should be set in the CMS for the largest breakpoint (i.e. 4 columns). So for one row enter 4, for two rows enter 8, for three rows enter 12, etc. The layout algorithm will work out how many items it should show for different breakpoints and adjust the number accordingly, appending a "View More" button if not all items are visible. For the smallest breakpoint (i.e. mobile devices), the row count is doubled.

Hope that makes sense, will leave this open until I've pushed the migration to update the relevant field descriptions in the CMS.
